iFixit/react-commerce
# Refactor product page featured products section

closes #1434 
closes #1206 

## QA

1. Open [product page preview](https://react-commerce-git-refactor-product-page-featured-dc6043-ifixit.vercel.app/products/repair-business-toolkit)
2. Verify that the featured products section adheres to [mockup](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=6650-80646&t=lm5sZ8asm7PGe46K-4) (Note: The title is the same of production, and the description in the default scenario is not shown. This is something that will be configurable when the section config will come from Strapi)

    <img width="1097" alt="Screenshot 2023-03-08 at 17 31 51" src="https://user-images.githubusercontent.com/4640135/223772338-e590cc10-b4b4-4e1e-abfe-5096904125d3.png">


   > 👁️ Note that on wider screens the section displays 6 columns to satisfy #1206 requirements

3. Verify that the section is responsive
